index.bs

```diff
@@ -1702,19 +1694,9 @@ urlPrefix: https://w3c.github.io/FileAPI/#dfn-; type: dfn;
 
 		1. Else
 
-			1. This is left to the implementation...
-
-				Issue: It's not good to leave things up to the
-				implementation. What should happen here?
+			1. Assign "" to |wellKnownFormat|.
```

Comment:
This seems like it could be a separate PR as it isn't directly related to allowing an optional sanitization of the well-known formats?

Reply:
Done.